roles/zuul/templates/etc/zuul/config/layout.yaml

```diff
@@ -67,6 +81,8 @@ projects:
       - ubuntu-hoist
     gate_github:
       - echo-true
+    experimental_github:
+      - ansible-syntax-BonnyCI/hoist
```

Comment:
because I'm not sure if it works, and I figured we would eventually want to add an experimental pipeline. but i suppose it's overkill for this job since our check pipeline isn't voting yet